documentation/cloud-manager-guide/admin/user-management/edit-user.md

```diff
@@ -0,0 +1,122 @@
+---
+keywords:
+title: Edit a User
+description: Edit a user
+---
+# Edit a User
+
+This guide outlines the required steps to edit the details, status, role, and environment assignments for a specific user in Environment Operations Center.
+
+## Getting started
+
+There are two ways to open the *Edit User* tab, either by selecting the user's name or by selecting the **Options** (**...**) menu associated with that user.
```

Comment:
Should probably mention here that they should be in the Admin section and are looking for the Users tab. Something like, "From the Users tab in the Admin section of EOC, there are two ways to edit a user." And is "Edit User" a tab? Or a screen? I think tabs are reserved for those that appear in the top navigation of a section vs a screen being anything else that opens up and a dialog being something that overlays the backing UI.